lib/ash_json_api/json_schema/json_schema.ex

```diff
@@ -246,9 +253,9 @@ defmodule AshJsonApi.JsonSchema do
 
       object =
         if links do
-          %{"data" => data, "links" => links}
+          %{"properties" => %{"data" => data, "links" => links}}
```

Comment:
Not sure if this is one of those swagger vs json-schema differences, but I had to wrap this map in an extra schema  object to get it to validate in swagger-editor.